pywbem/_logging.py

```diff
@@ -0,0 +1,402 @@
+# Copyright 2017 InovaDevelopment Inc. All Rights Reserved.
+
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+
+"""
+This package supports logging using the standard Python :mod:`py:logging`
+module. The logging support for multiple :class:`~py:logging.Logger` objects
+(named loggers):
+
+Currently there are two defined in this code:
```

Comment:
Remove "Currently". It is as defined right now, and we don't need to play that down.